CRM/Export/Form/Select.php

```diff
@@ -98,6 +98,7 @@ public function preProcess() {
     $components = array('Contact', 'Contribute', 'Member', 'Event', 'Pledge', 'Case', 'Grant', 'Activity');
 
     // FIXME: This should use a modified version of CRM_Contact_Form_Search::getModeValue but it doesn't have all the contexts
+    // FIXME: Or better still, use CRM_Core_DAO_AllCoreTables::getBriefName($daoName) to get the $entityShortName
```

Comment:
Ok, this is comment is unrelated to this PR... but I was looking for a function to get the Entity shortname and I found it!